pkg/apis/extensions/types.go

```diff
@@ -30,27 +30,26 @@ package extensions
 
 import (
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/resource"
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/util"
 )
 
-// ScaleSpec describes the attributes a Scale subresource
+// ScaleSpec describes the attributes of a Scale subresource
 type ScaleSpec struct {
-	// Replicas is the number of desired replicas. More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#what-is-a-replication-controller"
+	// Replicas is the number of desired replicas. Setting this number will set the number of the desired replicas in the underlying replication controller/deployment.
 	Replicas int `json:"replicas,omitempty"`
 }
 
 // ScaleStatus represents the current status of a Scale subresource.
 type ScaleStatus struct {
-	// Replicas is the number of actual replicas. More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#what-is-a-replication-controller
+	// Replicas is the number of actual replicas in the underlying replication controller/deployment.
 	Replicas int `json:"replicas"`
 
-	// Selector is a label query over pods that should match the replicas count. If it is empty, it is defaulted to labels on Pod template; More info: http://releases.k8s.io/HEAD/docs/user-guide/labels.md#label-selectors
+	// Selector is a label query over pods that should match the replicas count. More info: http://releases.k8s.io/HEAD/docs/user-guide/labels.md#label-selectors
 	Selector map[string]string `json:"selector,omitempty"`
 }
 
-// Scale subresource, applicable to ReplicationControllers and (in future) Deployment.
+// Scale subresource, applicable to ReplicationControllers and Deployments.
```

Comment:
I would just state the following:

"Scale subresource represents a scaling request for a resource."